### ISSUE TITLE
0 rounds and 10 players spam

### ISSUE THREAD

**Issue by danjono** [bugdone/headshotbox#42]
My console spam's out many demo's have 0 rounds and 10 players. This was not a problem until this bugfix https://github.com/bugdone/headshotbox/issues/41

Possibly related it is also coming up a new error that some demo's cannot be parshed.

Screenshot http://i.imgur.com/SJxIvI2.png
Logs https://www.dropbox.com/s/b6mkjgg1v4opyqw/headshotbox.rar?dl=0

**Comment by bugdone** [comment]
The spam happens whenever you go to the page of one of the players that played in one of these demos.
These *_900.dem demos are Valve tournament demos and don't work yet.

**Comment by danjono** [comment]
That makes sense, I do have some demo files from the Dreamhack tournament. Is it possible to have them ignored until they are implemented?

Also the spam happens when it is launched before I open the webpage. Is this correct?

**Comment by bugdone** [comment]
Yes, it happens just at startup, disregard the stuff I wrote above.
As for ignoring badly parsed demos, I'll add that to my todo list.
